fuzzers/041-clk-hrow-pips/top.py

```diff
@@ -0,0 +1,293 @@
+import os
+import re
+import random
+random.seed(int(os.getenv("SEED"), 16))
+from prjxray import util
+from prjxray.db import Database
+
+XY_RE = re.compile('^BUFHCE_X([0-9]+)Y([0-9]+)$')
```

Comment:
Is there a good place for this to be moved up as it is duplicated in fuzzers/040-clk-hrow-config/top.py

Reply:
This is one of those fine lines between DRY and clarity.  A general site name to X/Y function is probably useful. I'll think about it.